web/modules/custom/dpl_library_agency/dpl_library_agency.routing.yml

```
@@ -0,0 +1,17 @@
+dpl_library_agency.settings:
+  path: '/admin/config/dpl-library-agency'
+  defaults:
+    _controller: '\Drupal\system\Controller\SystemController::systemAdminMenuBlockPage'
+    _title: 'Library Agency'
+  requirements:
+    _permission: 'administer library agency configuration'
+
+dpl_library_agency.general_settings:
+  path: '/admin/config/dpl-library-agency/general-settings'
+  defaults:
+    _form: '\Drupal\dpl_library_agency\Form\GeneralSettingsForm'
+    _title: 'Dpl Library Agency General Settings'
```

```suggestion
    _title: 'General Settings'
```

I do not think we need to mention the module name in site administrator facing texts.

---

👍